Order.Management.PR/PaintingReport.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Order.Management
+{
+    class PaintingReport : Order
+    {
+        public int tableWidth = 73;
+        public PaintingReport(string customerName, string customerAddress, string dueDate, List<Shape> shapes)
+        {
+            base.CustomerName = customerName;
+            base.Address = customerAddress;
+            base.DueDate = dueDate;
+            base.OrderedBlocks = shapes;
+        }
+        public override void GenerateReport()
+        {
+            Console.WriteLine("\nYour painting report has been generated: ");
+            Console.WriteLine(base.ToString());
+            generateTable();
+        }
+
+        public void generateTable()
```

Comment:
`generateTable` method could be `private`. And for keeping the consistency, we should use pascal case naming, which is `GenerateTable`